exercises/mongo-shell/crud/read.md

```diff
@@ -10,16 +10,19 @@ A simple `count()` operation in order to count the number of documents in the co
 db.movieDetails.count()
 ```
 
-:bell:`count()` returns the total of all documents, which means duplicate documents will be counted.  
+:mushroom:`count()` returns the total of all documents, which means duplicate documents will be counted.  
```

Comment:
```suggestion
➡️ 
```

Reply:
Negative, you removed the whole line in favor of an ➡️.  did you mean:
```suggestion
:arrow_right:`count()` returns the total of all documents, which means duplicate documents will be counted.  
```